core/src/main/resources/application.ini

```diff
@@ -1,6 +1,6 @@
--J-Xms130m
+-J-Xms128m
```

Comment:
this doesnt seem right. You are going from 130m to 128m. same for the other two similar Xms below

Reply:
or are you just saying hey lets use computer rounded numbers?

Reply:
You are right, I wan to 2^n numbers :D

Reply:
xD 
No problems ;)